docs/language.md

```diff
@@ -0,0 +1,84 @@
+# The Language of Gutenberg
+
+At the core of Gutenberg lies the concept of the block. From a technical point of view, blocks both raise the level of abstraction from a single document to a collection of meaningful elements, and they replace ambiguity—inherent in HTML—with explicit structure. A post in Gutenberg is then a _collection of blocks_.
+
+To understand how blocks operate at a data-structure level, let's take a small detour to the simile of the printing press of Johannes Gutenberg. With the printing press, a “page” of a book was assembled from individual pieces and printed into a fully formed page. Once printed, there's no need to know it was built from multiple blocks of letters instead of one giant plate. In other words, the output is indifferent about how it was generated.
+
+This is true for content blocks. They are the way in which the user creates their content, but they no longer matter once the content is finished. That is, until it needs to be edited. Imagine if the printing press was able to print a page _while_ also including in the page the instructions to generate again the set of movable types required to print it. What we are doing with blocks could be compared to printing invisible marks on the text so that the printer can pick up, from an already printed page, the pieces it needs to reprint it.
```

Comment:
> What we are doing with blocks could be compared to printing invisible marks **in the margins** so that the printer can make adjustments to an already printed page without needing to set the page again from scratch.